client/components/tinymce/style.scss

```diff
@@ -198,6 +198,35 @@
 	border: 0;
 }
 
+.mce-toolbar .mce-wpcom-button.mce-contact-form button {
```

Comment:
Since many (most?) of these styles are identical to the media button, can we simply augment the media button selectors to include `.mce-contact-form`? 

```css
.mce-toolbar .mce-wpcom-button.mce-media button,
.mce-toolbar .mce-wpcom-button.mce-contact-form button {
	/* ... */
}
```